### PR TITLE
First commits

### PR DESCRIPTION
**Pull request by dav3r**
#  #

## 🗣 Description ##

This PR updates items inherited from the skeleton so that this repository can be used to successfully bootstrap a Domain Manager account with the `ProvisionAccount` role.

## 💭 Motivation and context ##

We can't do much of anything with this account until we bootstrap it.

Needed as part of https://github.com/cisagov/cool-system/issues/114. 

## 🧪 Testing ##

I followed the instructions in the `README` and verified that the `ProvisionAccount` role was successfully created in the Domain Manager (Staging) account.

## ✅ Checklist ##

* [x] This PR has an informative and human-readable title.
* [x] Changes are limited to a single goal - _eschew scope creep!_
* [x] All relevant type-of-change labels have been added.
* [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
* [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
* [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
* [x] All new and existing tests pass.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,38 +2,63 @@
...
+
+## Bootstrapping this account ##
+
+Note that the COOL Domain Manager account must be bootstrapped. This is
+because initially there is no IAM role that can be assumed to build out these
+resources. Therefore you must first apply the Terraform code using
+programmatic credentials for AWSAdministratorAccess as obtained for the
+COOL Domain Manager account from the COOL AWS SSO page.
+
+After this initial apply your desired IAM role will exist, and it will
```
Comment (hillaryj) [inline README.md]:
```suggestion
After this initial apply, your desired IAM role will exist, and it will
```

**Review by hillaryj — APPROVED** [review]
🎉

File: backend.tf [hunk 2]
```diff
@@ -0,0 +1,10 @@
+terraform {
+  backend "s3" {
+    encrypt        = true
+    bucket         = "cisa-cool-terraform-state"
```
Comment (felddy) [inline backend.tf:4]: I forget how this works.   Is this the actual bucket name?  Any issues if it is?

**Review by felddy — APPROVED** [review]
🍀 
Nice!  I'd like a sidebar about the state bucket at some point to refresh my brain.
💯

File: backend.tf (hunk 2 above)
Comment (jsf9k) [reply backend.tf:4]: It is the actual bucket name.  I have no issue with that.

**Review by jsf9k — APPROVED** [review]

**Pull request merged**